public/index.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="shortcut icon" href="<%=htmlWebpackPlugin.options.FAVICON_URL%>" type="image/x-icon" />
+    <script type="text/javascript" src="https://enterprise-tableau.edx.org/javascripts/api/tableau-2.min.js"></script>
```

Comment:
this script will be loaded for all but as per my understanding this should only load for the enterprise customers having tableau analytics enabled?

Reply:
added async